experimental/unitsconversion/src/info_provider.rs

```diff
@@ -0,0 +1,160 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// Provider structs must be stable
+#![allow(clippy::exhaustive_structs, clippy::exhaustive_enums)]
+
+//! Data provider struct definitions for this ICU4X component.
+//!
+//! Read more about data providers: [`icu_provider`]
+
+use alloc::borrow::Cow;
+use icu_provider::prelude::*;
+use zerovec::{VarZeroVec, ZeroMap};
+
+#[cfg(feature = "compiled_data")]
+#[derive(Debug)]
+/// Baked data
+///
+/// <div class="stab unstable">
+/// 🚧 This code is considered unstable; it may change at any time, in breaking or non-breaking ways,
+/// including in SemVer minor releases. In particular, the `DataProvider` implementations are only
+/// guaranteed to match with this version's `*_unstable` providers. Use with caution.
+/// </div>
+pub struct Baked;
+
+#[cfg(feature = "compiled_data")]
+const _: () = {
+    pub mod icu {
+        pub use crate as unitsconversion;
+    }
+    icu_unitsconversion_data::make_provider!(Baked);
+    icu_unitsconversion_data::impl_units_info_v1!(Baked);
+};
+
+#[cfg(feature = "datagen")]
+/// The latest minimum set of keys required by this component.
+pub const KEYS: &[DataKey] = &[UnitsInfoV1Marker::KEY];
```

Comment:
I have done this, I think we do not need the constants for now, because we will eliminate them.

However, I am still getting this error:

```
   Compiling icu_plurals v1.3.2 (/Users/younies/icu/icu4x/components/plurals)
   Compiling icu_decimal v1.3.2 (/Users/younies/icu/icu4x/components/decimal)
error[E0433]: failed to resolve: could not find `units_provider` in the crate root
  --> experimental/unitsconversion/src/provider.rs:33:5
   |
33 |     icu_unitsconversion_data::impl_units_info_v1!(Baked);
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `units_provider` in the crate root
   |
   = note: this error originates in the macro `icu_unitsconversion_data::impl_units_info_v1` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0433`.
error: could not compile `icu_unitsconversion` (lib) due to 5 previous errors
warning: build failed, waiting for other jobs to finish...
```